src/Compilers/CSharp/Portable/Symbols/Source/SourceDelegateMethodSymbol.cs

```diff
@@ -342,15 +356,18 @@ internal override OneOrMany<SyntaxList<AttributeListSyntax>> GetReturnTypeAttrib
 
         private sealed class EndInvokeMethod : SourceDelegateMethodSymbol
         {
-            private readonly RefKind refKind;
+            private readonly RefKind _refKind;
+            private readonly ImmutableArray<CustomModifier> _refCustomModifiers;
```

Comment:
>private readonly ImmutableArray<CustomModifier> _refCustomModifiers; [](start = 12, length = 68)

Consider replacing two these members with a reference to ```invoke```, then implementation of RefKind and RefCustomModifiers could simply delegate to the ```invoke```.